apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[resourceType]/index.vue

```diff
@@ -0,0 +1,185 @@
+<script setup lang="ts">
+import { IFilter, ISchemaMetaData, ITableMetaData } from "~/interfaces/types";
+const config = useRuntimeConfig();
+const route = useRoute();
+const router = useRouter();
+
+const pageSize = 10;
```

Comment:
bit longer?

Reply:
sure 20 ? could make it a (configurable)(overridable)global value, suggest nice clean separate pr for this

Reply:
not urgent for now indeed.